FirebaseAuth/Tests/Unit/PhoneAuthProviderTests.swift

```diff
@@ -818,7 +818,7 @@
       }
     }
 
-    class FakeAuthRecaptchaVerifier: AuthRecaptchaVerifier {
+    class FakeAuthRecaptchaVerifier: AuthRecaptchaVerifier, @unchecked Sendable {
```

Comment:
This looks okay for now since it's test code. What prompted this to be added though?

Reply:
CI failed for this asking to restate from sendable